### PR TITLE
feat: Pass group id (GID) in --user flag when calling docker run

### PR DESCRIPTION
**Pull request by manu-chroma**
Closes https://github.com/common-workflow-language/cwltool/issues/265

### CODE REVIEW

File: cwltool/docker_uid.py [hunk 1]
```diff
@@ -5,20 +5,20 @@
 from typing import List, Text
 
 
-def docker_vm_uid():  # type: () -> int
+def docker_vm_id():  # type: () -> int, int
```
Comment (mr-c) [inline cwltool/docker_uid.py]: `cwltool/docker_uid.py:8: error: syntax error in type comment`

**Comment by manu-chroma** [comment]
@mr-c Jenkins seems to be failing for some unknown reason. This PR seems ready now, can you review once again?

**Comment by mr-c** [comment]
```
Test failed: /var/lib/jenkins/jobs/cwltool-pr-conformance-multiver/workspace/version/v1.0/cwltool-venv/bin/cwltool --outdir=/tmp/tmp2h4c4L --quiet 'v1.0/revsort-packed.cwl#main' v1.0/revsort-job.json
Test compound workflow document
Returned non-zero
Got workflow error
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/cwltool/main.py", line 279, in single_job_executor
    r.run(**kwargs)
  File "build/bdist.linux-x86_64/egg/cwltool/job.py", line 394, in run
    euid, egid = docker_vm_id()
TypeError: 'NoneType' object is not iterable
Workflow error, try again with --debug for more information:
'NoneType' object is not iterable
```

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**